docs/content/how-to/aws/create-aws-hosted-cluster-arm-workers.md

```diff
@@ -41,6 +41,7 @@ spec:
     autoRepair: false
     upgradeType: Replace
   nodeDrainTimeout: 0s
+  nodeVolumeDetachTimeout: 0s
```

Comment:
Re-thinking this: I think it would be better to actually set the default to something here instead of keeping it as `0`.
That would be the cleanest way to leave it customizable for different use cases while still fixing a few issues for every consumer, without needing them to tweak something on their side. The main reason I didn't do that yet is that I don't know if we ever want to change that default back to 0, and how much effort it will be then (breaking change?).